staging/src/k8s.io/client-go/testing/fixture_test.go

```diff
@@ -322,3 +325,62 @@ func TestGetWithExactMatch(t *testing.T) {
 	errNotFound = errors.NewNotFound(gvr.GroupResource(), "pod")
 	assert.EqualError(t, err, errNotFound.Error())
 }
+
+func TestListPodAfterCreateSubresource(t *testing.T) {
```

Comment:
Please move this test to https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/kubernetes/typed/core/v1/fake/fake_pod_expansion_test.go